app/upgrades/v0_7_0/upgrades.go

```diff
@@ -15,13 +22,57 @@ const (
 var Upgrade = upgrades.Upgrade{
 	UpgradeName:          UpgradeName,
 	CreateUpgradeHandler: CreateUpgradeHandler,
+	StoreUpgrades: storetypes.StoreUpgrades{
+		Added: []string{
+			feegrant.StoreKey, feemarkettypes.StoreKey,
+		},
+		Renamed: nil,
+		Deleted: nil,
+	},
 }
 
 func CreateUpgradeHandler(
 	moduleManager *module.Manager,
 	configurator module.Configurator,
+	keepers *keepers.AppKeepers,
 ) upgradetypes.UpgradeHandler {
 	return func(ctx context.Context, plan upgradetypes.Plan, vm module.VersionMap) (module.VersionMap, error) {
-		return moduleManager.RunMigrations(ctx, configurator, vm)
+
+		vm, err := moduleManager.RunMigrations(ctx, configurator, vm)
+		if err != nil {
+			return vm, err
+		}
+
+		if err := ConfigureFeeMarketModule(ctx, keepers); err != nil {
+			return vm, err
+		}
+
+		return vm, nil
+	}
+}
+
+func ConfigureFeeMarketModule(ctx context.Context, keepers *keepers.AppKeepers) error {
+	sdkCtx := sdk.UnwrapSDKContext(ctx)
+
+	feeMarketParams, err := keepers.FeeMarketKeeper.GetParams(sdkCtx)
+	if err != nil {
+		return err
 	}
+
+	feeMarketParams.Enabled = true
+	feeMarketParams.FeeDenom = params.BaseCoinUnit
+	feeMarketParams.DistributeFees = true
+	feeMarketParams.MinBaseGasPrice = cosmosmath.LegacyMustNewDecFromStr("10")
```

Comment:
We could also set the default values for `Window` & `MaxBlockUtilization`: https://github.com/skip-mev/feemarket/blob/f1f216e15aa6bcc05075a7079496764820a0f545/x/feemarket/types/eip1559.go

Reply:
If we want to set these properties to the default values, we don't need to declare them here.